core/templates/dev/head/pages/signup-page/signup-page.controller.ts

```diff
@@ -18,84 +18,49 @@
 
 require('domain/utilities/UrlInterpolationService.ts');
 require('services/AlertsService.ts');
-require('services/CsrfService.ts');
+require('services/CsrfTokenService.ts');
 require('services/IdGenerationService.ts');
 require('services/SiteAnalyticsService.ts');
 require('services/UserService.ts');
 require('services/contextual/UrlService.ts');
 require('services/stateful/FocusManagerService.ts');
 
-oppia.controller('Signup', [
-  '$http', '$rootScope', '$scope', '$uibModal', 'AlertsService',
-  'CsrfService', 'FocusManagerService', 'SiteAnalyticsService',
-  'UrlInterpolationService', 'UrlService',
-  'SITE_NAME',
-  function(
-      $http, $rootScope, $scope, $uibModal, AlertsService,
-      CsrfService, FocusManagerService, SiteAnalyticsService,
-      UrlInterpolationService, UrlService,
-      SITE_NAME) {
-    var _SIGNUP_DATA_URL = '/signuphandler/data';
-    $rootScope.loadingMessage = 'I18N_SIGNUP_LOADING';
-    $scope.warningI18nCode = '';
-    $scope.siteName = SITE_NAME;
-    $scope.submissionInProcess = false;
-
-    $http.get(_SIGNUP_DATA_URL).then(function(response) {
-      var data = response.data;
-      $rootScope.loadingMessage = '';
-      $scope.username = data.username;
-      $scope.hasEverRegistered = data.has_ever_registered;
-      $scope.hasAgreedToLatestTerms = data.has_agreed_to_latest_terms;
-      $scope.showEmailPreferencesForm = data.can_send_emails;
-      $scope.hasUsername = Boolean($scope.username);
-      FocusManagerService.setFocus('usernameInputField');
-      CsrfService.setToken(data.csrf_token);
-    });
-
-    $scope.blurredAtLeastOnce = false;
-    $scope.canReceiveEmailUpdates = null;
-
-    $scope.isFormValid = function() {
-      return (
-        $scope.hasAgreedToLatestTerms &&
-        ($scope.hasUsername || !$scope.getWarningText($scope.username))
-      );
-    };
-
-    $scope.showLicenseExplanationModal = function() {
```

Comment:
Why have you deleted all this other stuff?

Reply:
They were deleted in another PR